README.md

```diff
@@ -87,6 +87,15 @@ compiling each contract in a separate process. To successfully run unit tests:
 2. Move the binary from `./target/release/zksolc` to a directory from `$PATH`, or add the target directory itself to `$PATH`.
 3. Run `cargo test`.
 
+## CLI testing
+
+For running command line interface tests, `zksolc` itself and `solc` must also be available in `$PATH`, because it calls itself recursively to allow compiling each contract with different commands. To successfully run CLI tests:
```

Comment:
`with different commands` -> `in separate processes`.

Reply:
updated as well